src/pages/concept-registration-page/form-concept/related-concepts/related-concepts-pure.component.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FC } from 'react';
+import { FieldArray } from 'formik';
+
+import { localization } from '../../../../lib/localization';
+import { HelpText } from '../../../../components/help-text/help-text.component';
+import { Can } from '../../../../casl/Can';
+import { AutosuggestConcepts } from '../../../../components/autosuggest-concepts/autosuggest-concepts.component';
+import { getTranslateText } from '../../../../lib/translateText';
+
+const addSeeAlso = (suggestion: string, form): void => {
+  const seeAlsoValues: string[] = form.values.seOgså;
+  seeAlsoValues.push(suggestion);
+  form.setFieldValue('seOgså', seeAlsoValues);
+};
+
+const removeSeeAlso = (index: number, form): void => {
+  const seeAlsoValues: string[] = form.values.seOgså;
+  seeAlsoValues.splice(index, 1);
+  form.setFieldValue('seOgså', seeAlsoValues);
+};
```

Comment:
Disse funksjonene kan tas bort hvis man bruker array helpers

Reply:
Jeg prøvde 'push' og 'remove' fra arrayHelpers, men det trigget ikke form validate til å kjøre (patch). Hvordan gjøre det da?

Reply:
Blir refaktorering sammen med med custom hook for å trigge patch.